frontend/src/components/Body.js

```diff
@@ -1,43 +1,75 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+import parse from 'html-react-parser';
+import './styles/BodyStyle.css';
 
 
 class Body extends Component {
     constructor(props) {
         super(props)
       
         this.state = {
-           get_child: []
+           get_child: [],
+           show_child: false, 
         }
+        this.handleClick = this.handleClick.bind(this)
       }
       componentDidMount(){
-          axios.get('http://localhost:4000/hawaBot/backend/get.php?id=1')
+         this.getChildrenByParentID(1)
+      }
+      getChildrenByParentID(parentID){
+        axios.get(`http://localhost:4000/hawaBot/backend/get.php?id=${parentID}`)
            
-            .then(response => {
-                console.log(response)
-                this.setState({get_child: response.data.children})
-            })
-            .catch(error => {
-                console.log(error)
-            })
+        .then(response => {
+            console.log(response)
+            this.setState({get_child: response.data.children})
+        })
+        .catch(error => {
+            console.log(error)
+        })
       }
+
+       handleClick() {
+       //console.log("Hello")
+       // if ({child.id} = 1){
+          this.setState (() => {
+            return {
+             show_child: !this.state.show_child 
+          }
+        });
+        
+     // }
+       
+         
+       
+       }
       
     render() {
         const {get_child} = this.state
         console.log("get_child:", get_child);
+       // const {show_child} = this.state.show_child;
```

Comment:
this should not be in a PR

Reply:
This file is no longer useful. I'll delete it